src/CommitmentsV2/SFA.DAS.CommitmentsV2/DependencyResolution/EncodingRegistry.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.Encoding;
+using StructureMap;
+
+namespace SFA.DAS.CommitmentsV2.DependencyResolution
+{
+    public class EncodingRegistry : Registry
+    {
+        public EncodingRegistry()
+        {
+            For<IEncodingService>().Use<EncodingService>();
```

Comment:
I think ideally this should be registered as a singleton as calling its constructor is relatively expensive.